### PR TITLE
Sb order confirmation/purchased items widget

### PR DESCRIPTION
**Pull request by Tim-M-S**
### All changes meet the following requirements
~~[ ] Changelog entry was added~~
- [x] Changes have been tested
- [x] Plugin can be built

@plentymarkets/ceres-io

### CODE REVIEW

File: resources/views/Widgets/Checkout/PurchasedItemsWidget.twig [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+{% set inlinePaddingT   = WidgetHelper.getInlineSpacings(spacingSettings, spacingSettings.customPadding.mobile, "", ["top"]) %}
+{% set spacingPaddingT  = WidgetHelper.getSpacingClasses(spacingSettings, spacingSettings.customPadding.mobile, "", ["top"]) %}
+
+<div class="widget widget-purchased-items widget-{{ widget.settings.appearance.mobile | default("primary") }}
+    {% if customClass | length > 0 %} {{ customClass }}{% endif %}
+    {% if spacingMargin | length > 0 %} {{ spacingMargin }}{% endif %}"
+    {% if inlineMargin | length > 0 %} style="{{ inlineMargin }}"{% endif %}>
+
+    <div class="confirmation-order-list">
+        {% for item in orderData.order.orderItems %}
```
Comment (cjohannsen) [inline resources/views/Widgets/Checkout/PurchasedItemsWidget.twig]: Bitte noch auf den TwigBuilder umbauen

**Review by cjohannsen — DISMISSED** [review]

File: resources/views/Widgets/Checkout/PurchasedItemsWidget.twig (hunk 1 above)
Comment (Tim-M-S) [reply resources/views/Widgets/Checkout/PurchasedItemsWidget.twig]: done

File: resources/views/Widgets/Checkout/PurchasedItemsWidget.twig [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+                <article class="cmp cmp-order-item">
+                    <a href="{{ Twig.print('data.itemURLs | getObjectValue(currentVariationId)') }}" class="item-image">
+                        <img src="{{ Twig.print('data.itemImages | getObjectValue(currentVariationId)') }}" alt="{{ Twig.print('item.orderItemName') }}" title="{{ Twig.print('item.orderItemName') }}" class="{% if spacingPaddingT | length > 0 %} {{ spacingPaddingT }} {% endif %}" {% if inlinePaddingT | length > 0 %} style="{{ inlinePaddingT }}"{% endif %}>
+                    </a>
+                    <div class="item-details small">
+                        <a href="{{ Twig.print('data.itemURLs | getObjectValue(currentVariationId)') }}" class="clearfix">
+                            <div class="item-name text-appearance {% if spacingPadding | length > 0 %}{{ spacingPadding }}{% endif %}"{% if inlinePadding | length > 0 %} style="{{ inlinePadding }}"{% endif %}>
+                                {{ Twig.set("orderItemName", Twig.call("ItemName.get", ["$item.orderItemName", "$item.bundleType"])) }}
+                                {{ Twig.print('orderItemName') }}
```
Comment (lks-mtzn) [inline resources/views/Widgets/Checkout/PurchasedItemsWidget.twig]:
```suggestion
                                {{ Twig.print(Twig.call("ItemName.get", ["$item.orderItemName", "$item.bundleType"])) }}
```

**Review by lks-mtzn — CHANGES_REQUESTED** [review]

File: resources/views/Widgets/Checkout/PurchasedItemsWidget.twig (hunk 2 above)
Comment (Tim-M-S) [reply resources/views/Widgets/Checkout/PurchasedItemsWidget.twig]: done

**Review by lks-mtzn — APPROVED** [review]

**Pull request merged**